### ISSUE TITLE
.Rproj icons not showing on Ubuntu

### ISSUE THREAD

**Issue by gorkang** [rstudio/rstudio#14626]
In a fresh installation of Rstudio in Ubuntu 24.04, the .Rproj files do not have the RStudio icons. 

There is a solution that has been working for me since Ubuntu 22.04:

FROM: https://askubuntu.com/a/1342095

```
sudo cp /usr/share/icons/hicolor/16x16/mimetypes/application-x-r-data.png /usr/share/icons/Yaru/16x16/mimetypes/
sudo cp /usr/share/icons/hicolor/16x16/mimetypes/application-x-r-project.png /usr/share/icons/Yaru/16x16/mimetypes/
sudo cp /usr/share/icons/hicolor/32x32/mimetypes/application-x-r-data.png /usr/share/icons/Yaru/32x32/mimetypes/
sudo cp /usr/share/icons/hicolor/32x32/mimetypes/application-x-r-project.png /usr/share/icons/Yaru/32x32/mimetypes/
sudo cp /usr/share/icons/hicolor/48x48/mimetypes/application-x-r-data.png /usr/share/icons/Yaru/48x48/mimetypes/
sudo cp /usr/share/icons/hicolor/48x48/mimetypes/application-x-r-project.png /usr/share/icons/Yaru/48x48/mimetypes/

sudo touch /usr/share/icons/Yaru
sudo gtk-update-icon-cache
# killall nautilus
```

### System details

    RStudio Edition : Desktop
    RStudio Version : 2024.04.0 Build 737
    OS Version      : Ubuntu 24.04 
    R Version       :  4.3.3

- [ x ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ x ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.posit.co/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@gorkang Thank you for raising and including the workaround! Note: I removed the 24.04 from the subject line to indicate that this is a problem in multiple versions of Ubuntu, not specific to 24.04.

**Comment by MatthieuStigler** [comment]
@ronblum is there any chance you upgrade this issue to a closer Milestone than 'Later'? It seems that the issue hasn't been addressed after more than 8 months? Thanks!

**Comment by gtritchie** [comment]
Putting into triage for discussion.

**Comment by MatthieuStigler** [comment]
hi @gtritchie and @ronblum this is admittedly not a big issue, but is there any chance it be given a higher priority? One year after, the problem is still there on Ubuntu 24.04.

Thanks!

**Comment by gtritchie** [comment]
Yeah, I'll bump this up and take a look. Should be a low-risk fix.

**Comment by gtritchie** [comment]
FYI, I screwed up when I did "Putting into triage for discussion" a year ago; I forgot to take it out of the Later milestone so it never appeared up in our triage view.

**Comment by kevinushey** [comment]
Confirmed fixed.

RStudio 2026.04.0-daily+313 "Globemaster Allium" Daily (aa4f6f63867b5df8befa49a5625b5cba6c9e9d3a, 2026-02-25) for Ubuntu Jammy
Mozilla/5.0 (X11; Linux aarch64) AppleWebKit/537.36 (KHTML, like Gecko) rstudio/2026.04.0-daily+313 Chrome/142.0.7444.265 Electron/39.6.0 Safari/537.36, Quarto 1.8.26

<img width="156" height="98" alt="Image" src="https://github.com/user-attachments/assets/92efe5eb-ddbe-49bc-8c7b-41d51b63397e" />
